ctapipe_io_nectarcam/tests/test_nectarcameventsource.py

```diff
@@ -44,3 +42,18 @@ def test_factory_for_nectarcam_file():
     from ctapipe_io_nectarcam import NectarCAMEventSource
     assert isinstance(reader, NectarCAMEventSource)
 
+def test_subarray():
+    from ctapipe_io_nectarcam import NectarCAMEventSource
+
+    n_events = 10
+    inputfile_reader = NectarCAMEventSource(
+        input_url=example_file_path,
+        max_events=n_events
+        )
+    subarray = inputfile_reader.subarray
+    subarray.info()
+    subarray.to_table()
+
+    n_camera_pixels = inputfile_reader.subarray.tel[0].camera.geometry.n_pixels
```

Comment:
We usually start at tel_id = 1, not 0

Reply:
For some unknown reason in NectarCam code I've seen so far tel_id = 0 is always used. Anyway this is just for the prototype camera for the moment so I do not think it makes a difference?

Reply:
The ctapipe instrument model should work no matter how you number your telescopes (even if you start at 100).  The difference is always made between `tel_id` (which is user-definable) and `tel_index` (the index into a flat array of telescopes).   If your first telescope ID is 0, then in this special case, `tel_id == tel_index` so everything should still work (though it also makes it easier for people to make index vs id mistakes.)

Reply:
In observed data not, but for simulated data there is an assumption that tel_ids start with 1 as this is needed to map true photo electrons to the correct telescope in simtel array files.

Reply:
Okay, let me test if setting this to 1 breaks anything, otherwise we can go for it.

Reply:
I do not manage to get a working version if I try to use tel_id=1. I'll leave this for later or for a ctapipe expert to look into.

Reply:
I mean it shouldn't matter here if it is 0 or 1, since this is not a simulation.  I tried it with this version, and it works fine with id=0.  Nevertheless, having it be id=1 may be less confusing later to users who are used to them starting at 1.  You can even try making it some larger number, to see if anyone made a bad assumption in your code :)

Reply:
Hi, I would suggest to the number that corresponds to the first deployed MST in the simulations of CTA-North